tests/attr/test_deeplift_basic.py

```diff
@@ -209,8 +213,12 @@ def gen_baselines_returns_array():
         assertTensorAlmostEqual(self, attributions[0], attributions_with_func[0])
         assertTensorAlmostEqual(self, attributions[1], attributions_with_func[1])
 
-    def test_relu_deepliftshap_with_custom_attr_func(self):
-        def custom_attr_func(multipliers, inputs, baselines):
+    def test_relu_deepliftshap_with_custom_attr_func(self) -> None:
+        def custom_attr_func(
+            multipliers: Tuple[Tensor, ...],
+            inputs: Tuple[Tensor, ...],
+            baselines: Tuple[Union[Tensor, int, float], ...],
```

Comment:
This type hint for baselines should actually be Tuple[Tensor, ...], since we internally convert to a Tensor before this method is called.